transaction/summary/src/verifier.rs

```diff
@@ -211,14 +250,15 @@ impl TxSummaryStreamingVerifierCtx {
         }
 
         // Now understand whose input this is. There are two cases
-        let entity = if tx_in_summary.input_rules_digest.is_empty() {
-            TransactionEntity::Ourself
+        if tx_in_summary.input_rules_digest.is_empty() {
+            // If we have no input rules digest, then this is a normal input
+            // add this to the report total
+            report.total_add(tx_in_summary_unblinding_data.into())?;
         } else {
-            TransactionEntity::Swap
+            // If we have input rules this is an SCI input and does not impact
+            // our balance
```

Comment:
removing the subtract entry here is why the "TODO: Fix this test" line linked above doesn't have a subtraction for the swap counterparty when they give MOB, at line 794 in `transaction/extra/tests/verifier.rs`